### ISSUE TITLE
Feedback on Apollo's Boons

### ISSUE THREAD

**Issue by Rosgath** [AlexKage69/OlympusExtra#90]
So I haven't seen anywhere to leave specific feedback on the boons themselves so I figured I'd just make one here and hopefully be better informed later.

1.) Blind is really strong. I recognize that it's consumed on a miss, but 40% chance to miss is very strong, especially on fast attack weapons like Malphon and Exagryph. I suggest reducing the miss chance, and a couple other changes. Here are some ideas:

1A.)Make it stacking. 3 should be enough, and have it remove one stack per miss. 

1B.) Make the miss chance variable based on the Slot it's in (lowest miss chance should be on your Attack, Medium and Special, and Highest on Cast and Revenge effect). Personally, I'd put it at 7%/Stack for Attack, 10%/stack for Special, 13% per stack for the Cast. Revenge would also be 13% per stack, but would also immediately apply 3 stacks.

2.) Apollo Call. The hit box on the beam is extraordinarily small. To the point that I often find myself missing enemies while the beam goes right over the target. I have to aim at their HP bars to reliably hit enemies, which feels weird. The damage granted for the Call is also pretty low considering it also restricts your movement, unlike any other Call.

2A.) Famed Duet is CRAZY strong, particularly against the final boss. His AI has a tendency to run and stand on Zag while he's using it, which means all 12 beams can hit him at once, racking up damage Impossibly fast. As in, I have had a run with full Zeus Bolts built very well (to the point that I can just hit Call for each phase and the call alone would drop him) and this did MORE damage than that with only 1 Pom at Epic. Thing is, **without** Famed Duet, Apollo call feels a bit lackluster.

2B.) I suggest either changing the Call to fire 3 beams by default (like [Redacted]'s first form of the attack) and/or double the width of the hitbox on the beams. I would also move the effect of Famed Duet to a Legendary Boon for Apollo (It's ok for him to have 2 legendaries I think.), and have another effect that makes the gods work better together (maybe something that works with Hunter's Mark? No one else works with that. No ideas on this one)

3.) Golden Fleece is REALLY cool, but is also around as much Money as Side Hussle, but sooner. This feels weird because Hermes Boons are balanced differently than those of other Gods. I feel this should be a bit lower(maybe 80, 100, 120?), or much lower and Trigger more often. (If there is a way, I would say it should trigger on the Laurel Rooms, encouraging players to go for harder rooms for bigger rewards) (Edit: I was just reviewing it in my head and I realize that Golden Fleece actually will generally give MORE Obols than Side Hussle, because Golden Fleece is affected by Gold multipliers from Poseidon/Chaos and Side Hussle is not. This means that if you got Golden Fleece and a Gold Multiplying Chaos boon both in Tartarus, you could very easily end up getting 1000+ Obols from this boon, which dwarfs the potential maximum of Side Hussle).

4.) I found Future Sight to be. . . one of those boons I actively avoided. That's not necessarily a bad thing (it's ok to have a boon be really good, but only in certain situations) but I just wanted to note that.

5.) Re: Sea Chantey. The word is Shanty. I don't know if you made it a pun on purpose, but I just thought I ought to point it out. Also, the boost feels kind of Low. Maybe 15%? Especially since Duo boons compete at the Epic/Legendary Quality level.

I did notice that Apollo doesn't really feel like he favors a particular Playstyle like the Other gods tend to do (As in, if you were to make a build of Only one god, they would play in a particular way). For instance, If I only took Dionysus I would focus on a High attack speed character who can stack up Hangover on as many enemies as possible, then drop Trippy Fog to finish things off as I saw them get low. Aphrodite focuses on going toe-to-toe with her enemies, getting Weak (and maximizing it) along with building up as much HP as she can to endure the hits. Poseidon Focuses on engaging enemies close to walls or pillars to maximize Knockback damage. Apollo feels like he lacks a particular Direction like that. Then again, I might have missed it.

Anyway, those are my thoughts on it. Overall it's really really good! I'm excited for Hephaestus!

**Comment by phypoh** [comment]
1) Blind was actually originally planned to be 50% and we nerfed it to 40%.
1A) We made the decision not to allow blind to be stackable because stacking actually buffs blind by a lot against fast attacking enemies (including bosses). This also buffs fast attacking weapons a lot, like fists. 
1B) Making miss chance based on the slot convolutes a lot of things, I'd rather not complicate the game mechanics if possible
2) Yeah the call needs a bit of polishing. We'll get to that after the Hestia release. Similar to earlier, Famed Duet actually used to be a lot more powerful to the point that you could burst down [REDACTED] on Phase 3 in 2 seconds XD But we've since nerfed the beams such that the damage from multiple beams don't stack as much. We also made Famed Duet really strong and fun in reference to Artemis/Apollo twin sibling enemy. I think we'll keep it as a duo instead of a legendary

5) We'll relook at balancing that. We were initially trying to figure out how to adjust this one, but decided to err on the low side and then see how people felt about it. Thanks for the feedback!

Apollo actually has a lot of different possible playstyles going for him. If you're on fists, you could go the dodge build which goes nicely with blind. You can opt to go cast build and stack casts with the Ares duo. You can also go Famed Duet build and then stack with Zeus boons. Think of Apollo as something like a luck-based Aphrodite, but with different duos. Try experimenting with Apollo a bit more, I think a few streamers have tried some interesting builds with Apollo

**Comment by AlexKage69** [comment]
I agree with @phypoh, but we really appreciate your feedback! And we'll do some changes in the next release to improve Apollo.
1) We'll increase the time between Blind being reapplied from 3.0sec to 4.0sec. This will make the Miss less OP.
2) The width of the beam will be increase as long as it's still visually appealing, since the hitbox is related to the visual fx. Also we'll nerf the multiple beam damage.
3) Golden Fleece gives Reroll based on Obol you gain, so you don't make money. But it goes well with Side Hussle if that's what you mean. Future Sight and Golden Fleece are balanced as it is from other feedback.
4) It's okay to have less popular boons since other people might like it.
5) "Sea Shanty" will be increase to 15%.
Thanks for the feedback!

**Comment by Rosgath** [comment]
So I miswrote when I said Golden Fleece (rerolls based on Obols collected). I meant Wishing Well (Bonus gold on Fountain use).

I did have a run recently where I rolled a +70% Obol Chaos Boon and picked up Wishing Well in Tartarus. It was. . . a lot if money. (250ish each biome, which ended up at just over 1k total from fountains). I did also beat Charon, so that made money go further though, so that may have been a factor on why it felt like so much.

Anyway, thanks for getting back to me! If I didn't feel like I'd hold you guys back with coding, I'd help. I'm still teaching myself C/C# though so maybe one day.
